src/wizard/javascript/index.md

```diff
@@ -4,28 +4,33 @@ doc_link: https://docs.sentry.io/error-reporting/quickstart/?platform=browser
 support_level: production
 type: language
 ---
+
 The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
 
 ```html
 <script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.apm.min.js"
```

Comment:
might be worth a small Note: callout that says they can view the full docs if they want to narrow the scope of the bundle

Reply:
@dcramer I searched the docs but couldn't find a list of bundles avail. Does that exist?

Reply:
Going to launch this and then update with a note once I know where to point it to.

Reply:
probably not cc @HazAT